hardcodes.R

```diff
@@ -121,4 +121,7 @@ issue_levels <- c("High Priority", "Error", "Warning")
 issue_display_cols <- c("Issue", "Type", "Guidance", "Detail")
 
 # EvaChecks data (contains issue, type, guidance for each check) ----------
-evachecks <- read_csv("public-resources/EvaChecks.csv", show_col_types = FALSE)
\ No newline at end of file
+evachecks <- read_csv("public-resources/EvaChecks.csv", show_col_types = FALSE)
+
+evachecks_no_dupes <- evachecks %>%
+  janitor::get_dupes(ID) %>% nrow() == 0
```

Comment:
is this just in case we don't deal with merge conflicts successfully or accidentally have two checks with the same ID?

In general, I think this brings up the more general question of how to catch/flag issues internally, without disrupting the app. Like this "evades" or "covers up" the issue. I think this is where testthat and stuff comes in. Like I'm thinking we should add a simple assertthat test that raises a warning in the console if there are any dupes.

I need to be much better about that sort of unit testing.

Reply:
YEP, that was the line of thinking, and there's definitely something missing here that needs to be expanded to be more meaningful, but I wanted to keep it in the script so we remember to look at it at least sometimes. I agree the test_that functions could really help us out here once we get those going